docker-compose-test.yml

```diff
@@ -5,7 +5,7 @@ services:
     image: localstack/localstack:1.3
     networks:
       - hmpps
-    container_name: localstack
+    container_name: localstack-sqs-2
```

Comment:
avoid name clash

Reply:
What's the name clashing with?

Reply:
"localstack" from the docker-composes of other projects